tests/superphot_plus/test_example_module.py

```diff
@@ -0,0 +1,13 @@
+from superphot_plus import example_module
```

Comment:
Can we change this to test one of the current files? It doesn't have to be an functionality test, just that we can import a module.  Something like 

```
from superphot_plus import constants

def test_constants() -> None:
    assert True
```